src/main/java/com/github/dockerjava/api/model/Statistics.java

```diff
@@ -19,6 +21,17 @@
     @JsonProperty("read")
     private String read;
 
+    /**
+     * @since Docker Remote API 1.21
+     */
+    @CheckForNull
+    @JsonProperty("networks")
+    private Map<String, Object> networksStats;
```

Comment:
IMHO this should be also implemented for every attribute ("cpu_stats", "memory_stats", etc), and not only for the new "networks". Maybe it is a good idea to close this PR and implement this enhancement in future patches.